tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -665,26 +647,26 @@ module.exports = {
       selector: '.files-table .oc-tbody-tr'
     },
     fileRowByResourcePathAnyType: {
-      selector: `//span[contains(@class, "oc-resource-name") and (@resource-name=%s or @resource-path=%s)]/ancestor::tr[contains(@class, "oc-tbody-tr")]`,
+      selector: `//span[contains(@class, "oc-data-test-resource-name") and (@data-test-resource-name=%s or @data-test-resource-path=%s)]/ancestor::tr[contains(@class, "oc-tbody-tr")]`,
```

Comment:
I'm a bit sceptic that `@class, "oc-data-test-resource-name"` isn't actually the class used here but I'll merge and, if necessary, revert the class name change :)